src/components/LinkButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+
+import { ButtonLink } from '../styles/dashboard';
```

Comment:
With the new package regading the relative path , shouldn't it be '/styles/' instead of '../styles/' ?